info_handler.py

```diff
@@ -0,0 +1,23 @@
+import glob
+
+from templates.module_search import ModuleSearch
+
+
+class InfoHandler(ModuleSearch):
```

Comment:
You may want to create a variable such as info_handler = InfoHandler() so that if multiple scripts want to use it, they don't have to each have to have their own InfoHandler.